sklearn/ensemble/forest.py

```diff
@@ -378,13 +378,16 @@ def feature_importances_(self):
 # ForestClassifier or ForestRegressor, because joblib complains that it cannot
 # pickle it when placed there.
 
-def accumulate_prediction(predict, X, out):
```

Comment:
The problem is that "out" is shared among threads, right?

Reply:
If this is the case I don't understand the memory increase. Shouldn't out be ``n_samples x n_classes`` and the memory increase therefore ``n_samples x n_classes x n_jobs``?